tests/memoryview/memoryview_no_binding_T3613.pyx

```diff
@@ -0,0 +1,11 @@
+# mode: compile
+# tag: memoryview
+
+# cython: binding=False
```

Comment:
Could be worth testing also the different `@binding()` settings on copies of the function below, to make sure they all work. (Assuming that doesn't accidentally make the test always pass without the fix.)

Reply:
I've added them (and checked that master doesn't pass this test with them added). I think it's only the module-level directive that's relevant, but it certainly doesn't hurt to test them.